bika/lims/browser/analysisrequest/analysisrequests.py

```diff
@@ -132,6 +131,12 @@ def __init__(self, context, request):
                 "attr": "getClientTitle",
                 "replace_url": "getClientURL",
                 "toggle": True}),
+            ("ClientID", {
+                "title": _("Client ID"),
+                "replace_url": "getClientURL",
+                "attr": "getClientID",
+                "replace_url": "getClientURL",
```

Comment:
Two times "replace_url". With one is enough :P

Reply:
Sorry, it should actually be the `index`